doc/source/deploy-guide/prepare-localhost.rst

```diff
@@ -112,6 +112,18 @@ configuration file loading ara plugins:
 For more details on ARA's web interface, please read
 https://ara.readthedocs.io/en/stable/webserver.html .
 
+Enable pipelining (recommended)
```

Comment:
If you revert the order, you need to adapt mitogen accordingly, as the content of mitogen doesn't go into [ssh_connection] section.